e2e-tests/cypress/tests/cypress.config.ts

```diff
@@ -2,6 +2,8 @@ import { defineConfig } from "cypress";
 import { addCucumberPreprocessorPlugin } from "@badeball/cypress-cucumber-preprocessor";
 import browserify from "@badeball/cypress-cucumber-preprocessor/browserify";
```

Comment:
```suggestion
import { addCucumberPreprocessorPlugin } from '@badeball/cypress-cucumber-preprocessor'
import browserify from '@badeball/cypress-cucumber-preprocessor/browserify'
```

Reply:
Sure, this will be handled in a seperate task to adapt the linting to our repo's standards.